integration-tests/api/__tests__/store/collections.js

```diff
@@ -87,4 +87,57 @@ describe("/store/collections", () => {
       })
     })
   })
+  describe("/store/collections?expand=products&fields=handle,name,description", () => {
```

Comment:
todo: Remove this and add the `it` on the same level as `it("lists collections", async () => {`.

You don't have to create a new describe.